test/entrypoint.sh

```diff
@@ -12,10 +12,19 @@ while ! exec 6<>/dev/tcp/0.0.0.0/3306; do
     sleep 1
 done
 
+# make sure we can reach the rabbitmq
+while ! exec 6<>/dev/tcp/0.0.0.0/5672; do
```

Comment:
Subsequent redirection of the same FD just replaces it, so this is fine:

```
$ echo foo > foo ; echo bar > bar ; cat 0<>foo 0<>bar
bar
```

I think the tidying below is just tidiness to make sure we're not holding a long-lived socket connection while the command runs.